cmd/ddev/cmd/cd.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/ddev/ddev/pkg/ddevapp"
+	"github.com/ddev/ddev/pkg/fileutil"
+	"github.com/ddev/ddev/pkg/globalconfig"
+	"github.com/ddev/ddev/pkg/output"
+	"github.com/ddev/ddev/pkg/util"
+	"github.com/spf13/cobra"
+	"path/filepath"
+)
+
+var bashFile = filepath.Join(globalconfig.GetGlobalDdevDir(), "commands/host/shells/ddev.sh")
+var fishFile = filepath.Join(globalconfig.GetGlobalDdevDir(), "commands/host/shells/ddev.fish")
+
+// CdCmd is the top-level "ddev cd" command
+var CdCmd = &cobra.Command{
+	Use:   "cd [project-name]",
+	Short: "Uses shell built-in 'cd' to change to a project directory",
+	Long: fmt.Sprintf(`To enable 'ddev cd' command, source the ddev.sh script from your rc-script.
+
+For bash:
+
+printf '\n[ -f "%s" ] && source "%s"' >> ~/.bashrc
+
+For zsh:
+
+printf '\n[ -f "%s" ] && source "%s"' >> ~/.zshrc
+
+For fish:
+
+echo \n'[ -f "%s" ] && source "%s"' >> ~/.config/fish/config.fish
+
+Restart your shell, and use 'ddev cd project-name'.
+`, bashFile, bashFile, bashFile, bashFile, fishFile, fishFile),
+	ValidArgsFunction: ddevapp.GetProjectNamesFunc("all", 1),
+	Example: `ddev cd
+command ddev cd project-name
```

Comment:
Without any explanation, this looks like a typo--at least it did to me until I saw the comments in `docs/content/users/usage/commands.md`.

Reply:
Thanks, I guess we don't need this example here (docs are enough).